src/screens/landing-page/landing-page.screen.tsx

```diff
@@ -1,20 +1,35 @@
+import { useHistory } from 'react-router-dom';
+import { Header } from 'components/common/auth-header.component';
 import { Box, Button, Image, Text } from 'components/primitive';
-
 import { WelcomeContainer, Title } from './landing-page.screen.styles';
 
 export const WelcomeScreen = () => {
+  let history = useHistory();
+
+  const redirectToLogin = () => {
+    history.push('/login');
```

Comment:
Use RoutePath here instead of hardcoding.
``
import { RoutePath } from 'navigation/route-path';
RoutePath.login
``

Reply:
done. pushed in the latest commit.